firefox/guide/firefox/firefox_preferences-background_data/oval/firefox.xml

```diff
@@ -0,0 +1,25 @@
+<def-group>
+  <definition class="compliance" id="firefox_preferences-background_data"  version="1">
+    <metadata>
+      <title>Background submission of information to Mozilla must be disabled.</title>
+      <affected family="unix">
+        <platform>Mozilla Firefox</platform>
+      </affected>
+      <description>There should be no background submission of technical and other information 
+	  from DoD computers to Mozilla with portions posted publically.</description>
```

Comment:
Let's make this usable for more people. Take out `from DoD computers`

Reply:
removed